command/daemon/metrics.go

```diff
@@ -0,0 +1,28 @@
+package daemon
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sirupsen/logrus"
+	"net/http"
```

Comment:
Move this import as the first one. And separate it from the rest with a blank line.